dev/bin/configure.sh

```diff
@@ -29,7 +29,6 @@ kubectl
 kustomize
 mvn
 ping
-rhoas
```

Comment:
is the code below, from this file still needed?
```
function configure_kafka {
  check_required_variable "MANAGED_KAFKA_INSTANCE_NAME"
  MANAGED_KAFKA_CREDENTIALS_FILE="${CREDENTIALS_FOLDER}/${MANAGED_KAFKA_INSTANCE_NAME}.json"
  ADMIN_SA_NAME="${MANAGED_KAFKA_INSTANCE_NAME}-admin"
  ADMIN_SA_CREDENTIALS_FILE="${CREDENTIALS_FOLDER}/${ADMIN_SA_NAME}.json"
  OPS_SA_NAME="${MANAGED_KAFKA_INSTANCE_NAME}-ops"
  OPS_SA_CREDENTIALS_FILE="${CREDENTIALS_FOLDER}/${OPS_SA_NAME}.json"
  MC_SA_NAME="${MANAGED_KAFKA_INSTANCE_NAME}-mc"
  MC_SA_CREDENTIALS_FILE="${CREDENTIALS_FOLDER}/${MC_SA_NAME}.json"
}
````

Reply:
I think it's ok to keep it